### ISSUE TITLE
Does not exit from mocha tests

### ISSUE THREAD

**Issue by Nogbit** [linkedin/sepia#8]
Your examples use ``process.exit(0)`` to finish/exit.  However, I can't do that in the middle of a mocha test started from gulp.  My tests run/pass and work great when cached but then just hang after the last line below.  What do you advise?

``VCR_MODE=cache gulp``

```Shell
[15:10:53] Using gulpfile /mmedia/documents/projects/uwscripts/nodemodules/node-uwsws/gulpfile.js
[15:10:53] Starting 'lint-src'...
[15:10:53] Starting 'lint-test'...
[15:10:53] Finished 'lint-test' after 98 ms
[15:10:53] Finished 'lint-src' after 319 ms
[15:10:53] Starting 'test'...

  
  ․․․․․․․․․․

  10 passing (38ms)

[15:10:54] Finished 'test' after 662 ms
[15:10:54] Starting 'default'...
[15:10:54] Finished 'default' after 5.43 μs
```

**Comment by Nogbit** [comment]
Fyi.... exporting ``httpServer`` server from `src/server.js` and then from ``index.js`` allows me to do the following in my outter most ``describe()`` and that works.  Shall I make a pull request?

```JavaScript
  after(function() {
    sepia.server.close();
  });
```

**Comment by avik-das** [comment]
I was just digging into this, and I found exactly what you pointed out. The fix is what you describe, but I'm going to look into exposing a cleaner API if I can, perhaps with some sort of `sepia.shutdown()` function. I don't want to just expose the internal server out into the open.

I'm happy to look at a pull request implementing that, or I'll work on it myself today.

**Comment by Nogbit** [comment]
Ok, that should do it unless maybe now or in the future you want the shutdown method to do more than what it's doing now.

**Comment by avik-das** [comment]
I published version 2.0.0. If you're not using the embedded server, then you can just upgrade, and you won't need to make any changes. If you do need it, you'll need to change your require to:

    var sepia = require('sepia').withSepiaServer();

And then call `sepia.shutdown()` whenever you need to shut it down.

**Comment by Nogbit** [comment]
That works!  Thanks for the quick change.
